packages/shared/src/components/tooltips/ContextMenuItem.tsx

```diff
@@ -33,7 +33,7 @@ function ContextMenuItem({
         className={classNames('typo-callout', className)}
         onClick={onClick}
       >
-        <span className="flex items-center w-full">
+        <span className=" flex w-full items-center">
```

Comment:
Just the extra space :P 
```suggestion
        <span className="flex w-full items-center">
```